src/pet.js

```diff
@@ -0,0 +1,68 @@
+function Pet(name) {
+    this.name = name;
+    this.age = 0;
+    this.hunger = 0;
+    this.fitness = 10;
+    this.children = [];
+}
+
+Pet.prototype = {
+    get isAlive() {
+        return this.age < 30 && this.hunger < 10 && this.fitness > 0;
+    }
+}
+
+Pet.prototype.growUp = function() {
+    if (!this.isAlive) {
+        throw new Error('Your pet is no longer alive :(');
+    };
+    const ADD_HUNGER = 5;
+    const TAKE_FITNESS = 3;
+    this.age += 1;
+    this.hunger += ADD_HUNGER;
+    this.fitness -= TAKE_FITNESS;
+}
+
+Pet.prototype.walk = function () {
+    if (!this.isAlive) {
+        throw new Error('Your pet is no longer alive :(');
+    }
+    const MAXIMUM_FITNESS = 10;
+    const ADD_FITNESS = 4;
+    (this.fitness + ADD_FITNESS) <= MAXIMUM_FITNESS ? this.fitness += ADD_FITNESS : this.fitness = MAXIMUM_FITNESS;
+}
+
+Pet.prototype.feed = function () {
+    if (!this.isAlive) {
+        throw new Error('Your pet is no longer alive :(');
+    }
+    const MIN_HUNGER = 0;
+    const TAKE_HUNGER = 3;
+    (this.hunger - TAKE_HUNGER) >= MIN_HUNGER ? this.hunger -= TAKE_HUNGER : this.hunger = MIN_HUNGER;
```

Comment:
As with the `walk` function, this could be refactored using the Math.max function:
```
this.hunger = Math.max(this.hunger - TAKE_HUNGER, MIN_HUNGER);
```